packages/teuchos/comm/src/Teuchos_StackedTimer.cpp

```diff
@@ -56,6 +56,12 @@ StackedTimer::LevelTimer::findBaseTimer(const std::string &name) const {
 BaseTimer::TimeInfo
 StackedTimer::LevelTimer::findTimer(const std::string &name, bool& found) {
   BaseTimer::TimeInfo t;
+  auto full_name = get_full_name();
+  auto level=0;
```

Comment:
level looks to be unused.

Reply:
you passed the test, very good...  (it was used before I realized my code could be much simpler)